src/api/routes/company.js

```diff
@@ -101,4 +102,45 @@ module.exports = (app) => {
                 throw err;
             }
         });
+
+    /**
+     * Enables a previously disabled company
+     */
+    router.put("/:companyId/enable",
+        or([
+            authMiddleware.isCompany,
+            authMiddleware.isAdmin,
+            authMiddleware.isGod
+        ], { status_code: HTTPStatus.UNAUTHORIZED, error_code: ErrorTypes.FORBIDDEN, msg: ValidationReasons.INSUFFICIENT_PERMISSIONS }),
+        validators.enable,
+        (req, res, next) => companyMiddleware.validCompany(req.params.companyId)(req, res, next),
+        async (req, res, next) => {
+            try {
+                await new OfferService().enableByCompany(req.params.companyId);
+                const company = await (new CompanyService()).enable(req.params.companyId);
+                return res.json(company);
+            } catch (err) {
+                return next(err);
+            }
+        });
+
+    /**
+     * Disables a previously enabled company
+     */
+    router.put("/:companyId/disable",
+        or([
+            authMiddleware.isCompany,
+            authMiddleware.isGod
```

Comment:
Can't an admin do this as well?

Reply:
If I recall correctly, the issue stated that companies and gods should be able to perform the disable operation, but mentioned nothing about admins, so I excluded them from being able to disable companies. Should I make it so admins can also disable companies?

Reply:
Oh, forgot about that. Ignore this suggestion then